src/app/modules/project/details/details.component.html

```diff
@@ -21,125 +21,104 @@
+            <p class="title">Likes <em class="fas fa-heart"></em></p>
+            <p (click)="likeClicked()">
+              <em [ngClass]="[project.userHasLikedProject ? 'fas' : 'far', this.animationTriggered ? 'animated' : '']" class="fa-heart"></em>
+              {{ project.likeCount }}
+            </p>
           </div>
-            <div class="project-metadata__item">
-              <p class='title'>Likes <em class="fas fa-heart"></em></p>
-              <p (click)="likeClicked()">
-                <em [ngClass]="[project.userHasLikedProject ? 'fas' : 'far', this.animationTriggered ? 'animated' : '']" class='fa-heart'></em>
-                {{project.likeCount}}
-              </p>
-            </div>
           <div class="project-metadata__item" [tooltip]="projectUpdated">
-            <p class='title cursor-pointer'>Upload Date <em class="fas fa-calendar"></em></p>
-            <p class='hover'>{{ project.created | date:'longDate' }}</p>
+            <p class="title cursor-pointer">Upload Date <em class="fas fa-calendar"></em></p>
+            <p class="hover">{{ project.created | date: "longDate" }}</p>
             <ng-template #projectUpdated>
-              <p>Created: {{project.created | date: 'mediumDate'}}</p>
-              <p>Updated: {{project.updated | date: 'mediumDate'}}</p>
+              <p>Created: {{ project.created | date: "mediumDate" }}</p>
+              <p>Updated: {{ project.updated | date: "mediumDate" }}</p>
             </ng-template>
           </div>
         </div>
 
         <ng-container *ngIf="isProduction()">
           <div class="project-tags">
-            <a class="project-tags__tag" href="javascript:void(0)">
-              Game
-            </a>
-            <a class="project-tags__tag" href="javascript:void(0)">
-              JavaScript
-            </a>
-            <a class="project-tags__tag" href="javascript:void(0)">
-              HTML
-            </a>
-            <a class="project-tags__tag" href="javascript:void(0)">
-              CSS
-            </a>
-            <a class="project-tags__tag" href="javascript:void(0)">
-              UX
-            </a>
-            <a class="project-tags__tag" href="javascript:void(0)">
-              Interaction
-            </a>
-            <a class="project-tags__see-more-link" href="javascript:void(0)">
-              see more tags&hellip;
-            </a>
+            <a class="project-tags__tag" href="javascript:void(0)"> Game </a>
+            <a class="project-tags__tag" href="javascript:void(0)"> JavaScript </a>
+            <a class="project-tags__tag" href="javascript:void(0)"> HTML </a>
+            <a class="project-tags__tag" href="javascript:void(0)"> CSS </a>
+            <a class="project-tags__tag" href="javascript:void(0)"> UX </a>
+            <a class="project-tags__tag" href="javascript:void(0)"> Interaction </a>
+            <a class="project-tags__see-more-link" href="javascript:void(0)"> see more tags&hellip; </a>
```

Comment:
@niraymak what's the status on the tags in the backend? Could you notify me whenever the frontend is ready to use the backend implementation for this? Thanks!

Reply:
I think Tim has reviewed and left some remark on @JVerbruggen PR. I'm still planning on reviewing it but had some higher priorities lately. Will review soon, maybe you can pick up Tim's feedback already? @JVerbruggen